From: Akiyoshi Kurita <weibu@redadmin.org>
Subject: docs: admin-guide/mm/damon: correct typo "directores"
Date: Sun, 28 Sep 2025 15:46:51 +0900

The word "directores" is a typo in usage.rst.  Correct it to "directories"
to improve documentation clarity.

Link: https://lkml.kernel.org/r/20250928064651.281325-1-weibu@redadmin.org
Signed-off-by: Akiyoshi Kurita <weibu@redadmin.org>
Reviewed-by: SeongJae Park <sj@kernel.org>
Signed-off-by: Andrew Morton <akpm@linux-foundation.org>
---

 Documentation/admin-guide/mm/damon/usage.rst |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/Documentation/admin-guide/mm/damon/usage.rst~docs-admin-guide-mm-damon-correct-typo-directores
+++ a/Documentation/admin-guide/mm/damon/usage.rst
@@ -360,7 +360,7 @@ The directory for the :ref:`quotas <damo
 DAMON-based operation scheme.
 
 Under ``quotas`` directory, four files (``ms``, ``bytes``,
-``reset_interval_ms``, ``effective_bytes``) and two directores (``weights`` and
+``reset_interval_ms``, ``effective_bytes``) and two directories (``weights`` and
 ``goals``) exist.
 
 You can set the ``time quota`` in milliseconds, ``size quota`` in bytes, and
_
